ryot_compass/src/bevy_compass/drawing/brush/geometric.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{Brush, BrushAction, DiamondBrush, RoundBrush, SquareBrush};
+use bevy::prelude::Reflect;
+use ryot::bevy_ryot::drawing::DrawingBundle;
+
+#[derive(Debug, Eq, PartialEq, Reflect, Copy, Clone, Hash)]
+pub enum GeometricBrush {
```

Comment:
This feels a bit "classy" to me, as in, abusing a bit too much of mutability.
Also, it seems like all of these have a size, and we expect them to because of the implementation requirements here. When would a geometric brush not have a size? Do we care to prepare this for a hypothetical future where we support non-regular shapes (i.e. rectangle)?

If not, I'd suggest turning this into a struct with a `size` and a `shape`, and keeping size public so you can be intentional from the outside about when this is going to be mutable vs not and behaviors are a bit more decooupled